cmd/pebble/cmd_validate_test.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2014-2020 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main_test
+
+import (
+	. "gopkg.in/check.v1"
+
+	pebble "github.com/canonical/pebble/cmd/pebble"
+)
+
+func (s *PebbleSuite) TestValidateCommand(c *C) {
+	_, err := pebble.Parser(pebble.Client()).ParseArgs([]string{"validate"})
+	c.Assert(err, IsNil)
```

Comment:
It looks like we're only testing the happy path here. Can we add a test of validation failure?

Reply:
Done.